nsenter/event.go

```diff
@@ -548,6 +566,14 @@ func (e *NSenterEvent) ReceiveResponse() *domain.NSenterMessage {
 	return e.ResMsg
 }
 
+func (e *NSenterEvent) ReapStartRequest() {
```

Comment:
I don't see this function called from anywhere; same for ReapEndRequest() below.

Reply:
You're right, i ended up not needing these new methods. Will remove them and merge.